docs/spelling_wordlist.txt

```diff
@@ -156,3 +156,5 @@ GroupType
 PathLike
 ElementType
 customizable
+sklearn
```

Comment:
for package name or suffix, maybe you can mention them by ``sklearn`` and ``csv`` rather than add them into spelling wordlist.

Reply:
used double backquotes to fix spellings not in wordlist